lib/command/src/Obelisk/Command/Run.hs

```diff
@@ -157,51 +173,52 @@ run root hackPaths = do
       , "Frontend.frontend"
       ]
 
-runRepl :: MonadObelisk m => FilePath -> PathTree HackOn -> m ()
-runRepl root hackPaths = do
-  pkgs <- getParsedLocalPkgs root hackPaths
+runRepl :: MonadObelisk m => FilePath -> PathTree Interpret -> m ()
+runRepl root interpretPaths = do
+  pkgs <- getParsedLocalPkgs root interpretPaths
   withGhciScript pkgs "." $ \dotGhciPath ->
     runGhciRepl root pkgs dotGhciPath
 
-runWatch :: MonadObelisk m => FilePath -> PathTree HackOn -> m ()
-runWatch root hackPaths = do
-  pkgs <- getParsedLocalPkgs root hackPaths
+runWatch :: MonadObelisk m => FilePath -> PathTree Interpret -> m ()
+runWatch root interpretPaths = do
+  pkgs <- getParsedLocalPkgs root interpretPaths
   withGhciScript pkgs root $ \dotGhciPath ->
     runGhcid root True dotGhciPath pkgs Nothing
 
-exportGhciConfig :: MonadObelisk m => FilePath -> PathTree HackOn -> m [String]
-exportGhciConfig root hackPaths = do
-  pkgs <- getParsedLocalPkgs root hackPaths
+exportGhciConfig :: MonadObelisk m => FilePath -> PathTree Interpret -> m [String]
+exportGhciConfig root interpretPaths = do
+  pkgs <- getParsedLocalPkgs root interpretPaths
   getGhciSessionSettings pkgs "."
 
-nixShellForHackPaths :: MonadObelisk m => Bool -> String -> FilePath -> PathTree HackOn -> Maybe String -> m ()
-nixShellForHackPaths isPure shell root hackPaths cmd = do
-  pkgs <- getParsedLocalPkgs root hackPaths
+nixShellForInterpretPaths :: MonadObelisk m => Bool -> String -> FilePath -> PathTree Interpret -> Maybe String -> m ()
+nixShellForInterpretPaths isPure shell root interpretPaths cmd = do
+  pkgs <- getParsedLocalPkgs root interpretPaths
   nixShellWithPkgs root isPure False (packageInfoToNamePathMap pkgs) shell cmd
 
 -- | Like 'getLocalPkgs' but also parses them and fails if any of them can't be parsed.
-getParsedLocalPkgs :: MonadObelisk m => FilePath -> PathTree HackOn -> m (NonEmpty CabalPackageInfo)
-getParsedLocalPkgs root hackPaths = parsePackagesOrFail =<< getLocalPkgs root hackPaths
+getParsedLocalPkgs :: MonadObelisk m => FilePath -> PathTree Interpret -> m (NonEmpty CabalPackageInfo)
+getParsedLocalPkgs root interpretPaths = parsePackagesOrFail =<< getLocalPkgs root interpretPaths
 
 -- | Relative paths to local packages of an obelisk project.
 --
 -- These are a combination of the obelisk predefined local packages,
 -- and any packages that the user has set with the @packages@ argument
 -- to the Nix @project@ function.
-getLocalPkgs :: forall m. MonadObelisk m => FilePath -> PathTree HackOn -> m (Set FilePath)
-getLocalPkgs root hackPaths = do
-  putLog Debug [i|Finding packages with root ${root} and hacking paths ${hackPaths}|]
+getLocalPkgs :: forall m. MonadObelisk m => FilePath -> PathTree Interpret -> m (Set FilePath)
+getLocalPkgs root interpretPaths = do
+  putLog Debug $ [i|Finding packages with root ${root} and interpret paths:|] <> "\n" <> drawPathTree textInterpret interpretPaths
```

Comment:
You can just inline the expr into the QQ

Reply:
I think?

Reply:
I did, but the newline didn't work.